### PR TITLE
Fixes columns being split by selected sections; Fix #118

### LINKED ISSUE #118: Sorting deck by columns with item selected splits the selection
See #56 for specifics

### PR DESCRIPTION
**Pull request by ctrlaltca**
Fix selection problem when sorting the deck list

### CODE REVIEW

**Comment by ZeldaZach** [comment]
Works fully as expected and is a minor fix. Merging :)

File: cockatrice/src/decklistmodel.cpp [hunk 1]
```diff
@@ -311,9 +311,9 @@ void DeckListModel::sortHelper(InnerDecklistNode *node, Qt::SortOrder order)
         const int fromRow = sortResult[i].first;
         const int toRow = sortResult[i].second;
         AbstractDecklistNode *temp = node->at(toRow);
-        for (int j = columnCount(); j; --j) {
-            from << createIndex(fromRow, 0, temp);
-            to << createIndex(toRow, 0, temp);
+        for (int j = 0; j < columnCount(); ++j) {
```
Comment (Psithief) [inline cockatrice/src/decklistmodel.cpp]: While this won't have much effect in this instance, I think calling `columnCount()` every loop is considered bad practice.

Comment (ctrlaltca) [inline cockatrice/src/decklistmodel.cpp]: agreed

Comment (ctrlaltca) [inline cockatrice/src/decklistmodel.cpp]: Fixed in #928

**Pull request merged**